tools/ImportBuddy/source/ImportBuddy/TheDiscDb.MakeMkv/LogParser/LogLines/LogLine.cs

```diff
@@ -37,6 +37,19 @@ protected static int TryParseInt(int index, string[] parts)
             return default(int);
         }
 
+        protected static int TryParseInt(CsvEnumerator enumerator)
```

Comment:
Should these `TryParse*(CsvEnumerator enumerator)` methods just be methods on the CsvEnumerator? Or if not that than maybe extension methods? They seem out of place here

Reply:
Yeah, those can be moved to CsvEnumerator, was just trying to avoid unnecessary changes to the structure of things.